### PR TITLE
Fix AppSwitcher currentApp Icon Size

### PR DESCRIPTION
**Pull request by RychenerLorenz**
The icons from the portal can be too big. Limiting them to wight 100% should help.

### CODE REVIEW

File: lib/components/app-switcher/AppSwitcher.vue [hunk 1]
```diff
@@ -13,8 +13,8 @@
           :style="{ 'background-color': hover || menuOpen ? '' : currentAppColor }"
           v-on="on"
         >
-          <v-icon v-if="hasApps && (hover || menuOpen)" color="primary"> mdi-apps </v-icon>
-          <img v-else-if="currentApp" :src="base64ImageURI(currentApp.svg_icon)" />
+          <v-icon v-if="hasApps && (hover || menuOpen)" width="100%" color="primary"> mdi-apps </v-icon>
```
Comment (all4manu) [inline lib/components/app-switcher/AppSwitcher.vue:16]: I think this here wouldn't work. `v-icon` doesn't have a `width` property but a `size`. Do you really need to define a size here. By default Vuetify uses `24px` for the size.

File: lib/components/app-switcher/AppSwitcher.vue [hunk 2]
```diff
@@ -13,8 +13,8 @@
...
+          <img v-else-if="currentApp" width="100%" :src="base64ImageURI(currentApp.svg_icon)" />
```
Comment (all4manu) [inline lib/components/app-switcher/AppSwitcher.vue:17]: This is not valid HTML. The `width` attribute is defined as: `The intrinsic width of the image in pixels. Must be an integer without a unit.`.

**Review by all4manu — CHANGES_REQUESTED** [review]

File: lib/components/app-switcher/AppSwitcher.vue (hunk 2 above)
Comment (all4manu) [reply lib/components/app-switcher/AppSwitcher.vue:17]: How about adding a `style="max-height: 50px; max-width: 50px"`?

File: lib/components/app-switcher/AppSwitcher.vue (hunk 1 above)
Comment (RychenerLorenz) [reply lib/components/app-switcher/AppSwitcher.vue]: 👍

**Review by all4manu — APPROVED** [review]

**Pull request merged**